packages/emails/src/templates/OrganizerScheduledEmail.tsx

```diff
@@ -26,10 +27,10 @@ export const OrganizerScheduledEmail = (
     title = "new_event_scheduled";
   }
 
-  const t = props.calEvent.organizer.language.translate;
+  const t = props.teamMember?.language.translate || props.calEvent.organizer.language.translate;
   return (
     <BaseScheduledEmail
-      timeZone={props.calEvent.organizer.timeZone}
+      timeZone={props.teamMember?.timeZone || props.calEvent.organizer.timeZone}
```

Comment:
By passing the team member we can personalize the language and time zone in the email